internal/provider/provider_test.go

```diff
@@ -40,7 +40,7 @@ func init() {
 			nextProvider := providerserver.NewProtocol5(NewFrameworkProvider())
 
 			mux, err := tf5muxserver.NewMuxServer(
-				ctx, nextProvider, PluginProviderServer, testAccProvider.GRPCProvider,
```

Comment:
I had lots of test failures unless `testAccProvider.GRPCProvider` was included in this list

Reply:
Thanks, added back in c954bff5.